sorc/grid_tools.fd/regional_esg_grid.fd/regional_esg_grid.f90

```diff
@@ -2,7 +2,7 @@
 !! @brief ???
```

Comment:
I can fill in this ???

Reply:
OK, check out the branch or just type it here...